SU-HSDO/suhumsci
# styles for all people fields

# READY FOR REVIEW

# Summary
- Added necessary styles for all fields on people content type

# Needed By (Date)
- End of sprint (8/16)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Navigate to person node that is fully populate (even though this should never happen since there are fields for staff, faculty, students, etc.)
3. Ensure that styles and collection of items together look good

# Affected Projects or Products
- Archaeology
- suhumsci (theme)

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/ARCH-319
- Need to pull new display settings from https://swshumsci-stage.stanford.edu/admin/structure/types/manage/hs_person/display-layout/default

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
